tekton/pipeline_run_test.go

```diff
@@ -41,15 +43,22 @@ type ReleasePipelineParams struct {
 }
 
 const (
-	PipelineRunPrefixName = "mypipeline"
+	PipelineRunPrefixName = "test-pipeline"
```

Comment:
You probably don't need to export these variables. These could cause you problems if other tests under the same package.